src/10-webpage-publishing-with-ipfs/index.md

```diff
@@ -0,0 +1,309 @@
+
+## What's next? More details and improvements
+
+(recap of what we just did)
+
+Let's zoom out and take look at what we just did:
+
+0. We installed the `ipfs` program and ran an IPFS node.
+1. We created a simple website. It could be much more elaborate, but the steps would be the same.
+2. We published the website to the IPFS network (with `ipfs add`), making it viewable to the whole network.
+3. We explored `ipscend`, a tool to version and publish websites with ipfs.
+4. We explored DNS naming with `ipfs` and `dnslink`, a tool to publish special `TXT` records.
+5. And, finally, we published content to the peer-to-peer web, in a future-safe, resilient way, while depending (mostly) on only one thing: the IPFS protocol.
+
+When it comes to publishing web applications and web sites, some of the requirements to optimize the workflow and enhance the developer experience are very similar to the ones for publishing code packages. We've talked and demonstrated how IPFS is a perfect transport for moving around [packages of code](https://www.youtube.com/watch?v=-S-Tc7Gl8FM) or [container-ized services](https://www.youtube.com/watch?v=vaIWRyotz4g), Mainly, this is due to its ability to use bandwidth very efficiently with a very smart and distributed discovery mechanism, which guarantees integrity for the content being looked up.
+
+There is much more to do, more problems to simplify, and lots to implement. We will end this post highlighting a some things we're working on to improve the web publishing experience even further.
+
+#### Timeline view
+
+One other feature that will significantly increase the developer experience with `ipscend` is [**versioning**](https://github.com/ipfs/notes/issues/23). Apps, as with any software, have several iterations across their lifetime. These iterations pack different things, like new features, complete application revamps or sometimes, regressions.
+
+Version Control Systems have enabled developers to work collaboratively on the same code, without wasting time dealing with merge conflicts. We can improve these tools by letting developers, designers and other individuals have more power. Ultimately, they should be able to have quick access to the timeline for their application, to make annotations on the current iterations before a release, to cherry pick which version to be released (using visual rollbacks), to analyze if there have been any regressions in a specific browser, among other things.
```

Comment:
This paragraph is really information dense. Break it up for each thing web developers could do, and talk about how our efforts there are going.